docs/content/index.fsx

```diff
@@ -1,88 +1,10 @@
-(*** hide ***)
-// This block of code is omitted in the generated HTML documentation. Use 
-// it to define helpers that you do not want to show in the documentation.
-#I "../../bin"
-
 (**
 Kafunk - F# Kafka client
 ======================
 
-Example
--------
-
-This example demonstrates a few uses of the Kafka client.
-
-*)
-
-#r "kafunk.dll"
-open Kafunk
-
-let conn = Kafka.connHost "existentialhost"
-
-
-let metadata = 
-  Kafka.metadata conn (Metadata.Request([|"absurd-topic"|])) 
-  |> Async.RunSynchronously
-
-for b in metadata.brokers do
-  printfn "broker|host=%s port=%i nodeId=%i" b.host b.port b.nodeId
-
-for t in metadata.topicMetadata do
-  printfn "topic|topic_name=%s topic_error_code=%i" t.topicName t.topicErrorCode
-  for p in t.partitionMetadata do
-    printfn "topic|topic_name=%s|partition|partition_id=%i" t.topicName p.partitionId
-
-
-
-let producerCfg = 
-  ProducerCfg.create ([|"absurd-topic"|], Partitioner.konst 0, requiredAcks=RequiredAcks.Local)
-
-let producer = 
-  Producer.createAsync conn producerCfg 
-  |> Async.RunSynchronously
+Kafunk is a [Kafka](http://kafka.apache.org/) client written in F# and provides an idiomatic F# API.
```

Comment:
We have a ways to go with the API but I think it's fair to say this is a goal of ours.